setup-travis.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+set -ev
+
+gem update --system
+gem --version
+#cache file containing vendor/bundle/cache - to speed up bundle instann and to fix possible connection itmeouts to rubygems
+wget -qO- http://testmycode.net/travis/2.2.0-tmc-server-bundler-cache.tar.gz | tar xz
+BUNDLE_GEMFILE=Gemfile bundle install --retry=3 --jobs=3 --deployment
```

Comment:
Maybe comment why setting BUNDLE_GEMFILE explicitly (here and below) is necessary?